test/zippy/mzcompose.py

```diff
@@ -155,11 +159,15 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             },
         ),
         Materialized(
-            external_minio=True,
-            external_cockroach=True,
+            external_minio="toxiproxy",
```

Comment:
You need to do the same in `misc/python/materialize/zippy/mz_actions.py` , line 70.

Reply:
Ops sorry, you did that already.